kolibri/core/assets/src/views/core-base.vue

```diff
@@ -110,6 +115,7 @@
       },
     },
     mounted() {
+      this.updateDocumentTitle(this.title);
```

Comment:
Yeah - this relates to getting string wrapping in JS files, because titles are generated in actions. I'd started working on that a little but that got distracted.